kasmvnc/run.sh

```diff
@@ -14,166 +15,167 @@ check_installed() {
 
 # Function to download a file using wget, curl, or busybox as a fallback
 download_file() {
-  local url=$1
-  local output=$2
-  if command -v wget &> /dev/null; then
-    wget $url -O $output
-  elif command -v curl &> /dev/null; then
-    curl -fsSL $url -o $output
+  local url="$1"
+  local output="$2"
+  local download_tool
+
+  if command -v curl &> /dev/null; then
+    download_tool="curl -fsSL"
+  elif command -v wget &> /dev/null; then
+    download_tool="wget -q -O-"
   elif command -v busybox &> /dev/null; then
-    busybox wget -O $output $url
+    download_tool="busybox wget -O-"
   else
-    echo "Neither wget, curl, nor busybox is installed. Please install one of them to proceed."
+    echo "ERROR: No download tool available (curl, wget, or busybox required)"
     exit 1
   fi
-}
 
-# Function to install kasmvncserver for debian-based distros
-install_deb() {
-  local url=$1
-  download_file $url /tmp/kasmvncserver.deb
-  sudo apt-get update
-  DEBIAN_FRONTEND=noninteractive sudo apt-get install --yes -qq --no-install-recommends --no-install-suggests /tmp/kasmvncserver.deb
-  sudo adduser $USER ssl-cert
-  rm /tmp/kasmvncserver.deb
+  $download_tool "$url" > "$output" || {
+    echo "ERROR: Failed to download $url"
+    exit 1
+  }
 }
 
-# Function to install kasmvncserver for Oracle 8
-install_rpm_oracle8() {
-  local url=$1
-  download_file $url /tmp/kasmvncserver.rpm
-  sudo dnf config-manager --set-enabled ol8_codeready_builder
-  sudo dnf install oracle-epel-release-el8 -y
-  sudo dnf localinstall /tmp/kasmvncserver.rpm -y
-  sudo usermod -aG kasmvnc-cert $USER
-  rm /tmp/kasmvncserver.rpm
+# Add user to group using available commands
+add_user_to_group() {
+  local user="$1"
+  local group="$2"
+
+  if command -v usermod &> /dev/null; then
+    sudo usermod -aG "$group" "$user"
+  elif command -v adduser &> /dev/null; then
+    sudo adduser "$user" "$group"
+  else
+    echo "ERROR: At least one of 'adduser'(Debian) 'usermod'(RHEL) is required"
+    exit 1
+  fi
 }
 
-# Function to install kasmvncserver for CentOS 7
-install_rpm_centos7() {
```

Comment:
I wasn't looking at the documentation for the install, but rather the release page.
There is no release for Centos, at least not for v1.3.2.
I do see it was compiled for v1.3.1.

I'm assuming that their builds are fully automated, and they are dropping support for Centos?

Reply:
It doesn't look like they've intentionally dropped CentOS, but can't say what's going on there. They never added support for CentOS 8 even though an issue has been open since 2020. And "just" last year they fixed a build issue with CentOS 7, and no updates or removals since then `¯\_(ツ)_/¯`.

It's also still in the official docs: https://www.kasmweb.com/kasmvnc/docs/latest/install.html#centos-oracle-7

Reply:
I think we can ignore it for now and add support if someone needs it?
I would imagine most people are going to user the enterprise-desktop image anyways.

Either that or an official KasmWeb image, which is still a WIP.